index.js

```diff
@@ -194,6 +227,28 @@ class DetailsDialogElement extends HTMLElement {
     if (!details) return
     toggleDetails(details, open)
   }
+
+  static get observedAttributes() {
+    return ['src', 'preload']
+  }
+
+  attributeChangedCallback() {
+    const details = this.parentElement
+    if (!details) return
+    const state = initialized.get(this)
+    if (!state) return
+
+    if (this.src) {
+      details.addEventListener('toggle', loadIncludeFragment, {once: true})
+
+      if (this.preload) {
+        details.addEventListener('mouseover', loadIncludeFragment, {once: true})
+      }
+    } else {
+      details.removeEventListener('toggle', loadIncludeFragment)
+      details.removeEventListener('mouseover', loadIncludeFragment)
```

Comment:
I double checked the [documentation](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#Matching_event_listeners_for_removal), and removing a listener without passing the extra parameters, like `once: true`, works fine. 📚